index.html

```diff
@@ -1933,6 +1933,24 @@ <h3 id="Error Messages" style="left: -46px; position: relative"> Error Messages<
         longer error messages and helps keep the 100-character limit on lines as mentioned earlier in the
         style guide </p>
   </div>
+  <!-- Production Tests -->
+  <h3 id="Production Tests" style="left: -46px; position: relative"> Production Tests</h3>
+  <div class="stylebody">
+    <p> Production tests may be resource intensive and may take some time to run; these are categorised separately from other tests. Production tests do not run when the user enters ‘make test’ (being non-important and/or expensive in execution time). 
```

Comment:
Quick fix: 'make test' runs all tests, including production tests, make test-production runs exclusively the production tests. I'd also add a sentence mentioning make test-production : )

Reply:
Also stress the importance of adding '_prod' and not 'prod' or 'production'. As the makefile only recognises file's ending in _prod as production and nothing else : ).